### PR TITLE
Add patches for sailfish and sailjail support

### PR DESCRIPTION
**Pull request by spiiroin**

### CODE REVIEW

File: rpm/0002-Fix-symlinks-that-go-though-proc-self.patch [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
++	size_t pfix = strlen(proc_pid);
++	if (strncmp(use_target, proc_pid, pfix))
++		goto done;
++
++	if( use_target[pfix] != 0 && use_target[pfix] != '/' )
++		goto done;
++
++	char *tmp;
++	if( asprintf(&tmp, "%s%s", proc_self, use_target + pfix) != -1 ) {
++		fprintf(stderr, "SYMLINK %s\n  -->   %s\n", use_target, tmp);
```
Comment (spiiroin) [inline rpm/0002-Fix-symlinks-that-go-though-proc-self.patch:60]: These are now emitted unconditinally, via fprintf() - probably should be using something that is enabled on --debug only?

Comment (monich) [reply rpm/0002-Fix-symlinks-that-go-though-proc-self.patch:60]: IMO it should be off by default and turned on with `--debug`. At very least it should be silenced with `--quiet`

Comment (spiiroin) [reply rpm/0002-Fix-symlinks-that-go-though-proc-self.patch]: Looks like there is not really a common"logging api" in firejail -> made the output conditional based on arg_debug which gets set based on --debug. And handled debug logging for "protocol" option in similar manner.

**Comment by spiiroin** [comment]
Some for/while/if style fixes. Debug logging enabled with --debug option. Comma separated list filter now accepts also "=item" to reset list and just to have symmetry regarding "-item",  "+litem" is treated similarly to "item".

**Comment by rainemak** [comment]
I guess this is ok now?

**Comment by rainemak** [comment]
Can be improved more later and upstreaming can be done after this,

**Review by rainemak — APPROVED** [review]
Approving...

**Pull request merged**